tests/core/test_fileio.py

```diff
@@ -21,80 +21,85 @@
+                    'dropout': 0.0,
+                    'residual': False,
+                    'separable': True,
+                    'se': True,
+                    'se_context_size': -1,
+                }
+            ],
+        },
+    }
+
+    decoder = {
+        'cls': 'nemo.collections.asr.modules.ConvASRDecoder',
+        'params': {
+            'feat_in': 1024,
+            'num_classes': 28,
+            'vocabulary': [
+                ' ',
+                'a',
+                'b',
+                'c',
+                'd',
+                'e',
+                'f',
+                'g',
+                'h',
+                'i',
+                'j',
+                'k',
+                'l',
+                'm',
+                'n',
+                'o',
+                'p',
+                'q',
+                'r',
+                's',
+                't',
+                'u',
+                'v',
+                'w',
+                'x',
+                'y',
+                'z',
+                "'",
+            ],
+        },
+    }
+    modelConfig = DictConfig(
+        {'preprocessor': DictConfig(preprocessor), 'encoder': DictConfig(encoder), 'decoder': DictConfig(decoder)}
+    )
+
+    return EncDecCTCModel(cfg=modelConfig)
 
-        decoder = {
-            'cls': 'nemo.collections.asr.modules.ConvASRDecoder',
-            'params': {
-                'feat_in': 1024,
-                'num_classes': 28,
-                'vocabulary': [
-                    ' ',
-                    'a',
-                    'b',
-                    'c',
-                    'd',
-                    'e',
-                    'f',
-                    'g',
-                    'h',
-                    'i',
-                    'j',
-                    'k',
-                    'l',
-                    'm',
-                    'n',
-                    'o',
-                    'p',
-                    'q',
-                    'r',
-                    's',
-                    't',
-                    'u',
-                    'v',
-                    'w',
-                    'x',
-                    'y',
-                    'z',
-                    "'",
-                ],
-            },
-        }
-        modelConfig = DictConfig(
-            {'preprocessor': DictConfig(preprocessor), 'encoder': DictConfig(encoder), 'decoder': DictConfig(decoder)}
-        )
-        asr_model = EncDecCTCModel(cfg=modelConfig)
+
+class FileIOTest:
+    @pytest.mark.unit
+    def test_to_from_config_file(self, asr_model):
 
         with tempfile.NamedTemporaryFile() as fp:
             yaml_filename = fp.name
             asr_model.to_config_file(path2yaml_file=yaml_filename)
             next_instance = EncDecCTCModel.from_config_file(path2yaml_file=yaml_filename)
-            self.assertTrue(isinstance(next_instance, EncDecCTCModel))
+            assert isinstance(next_instance, EncDecCTCModel) == True
```

Comment:
Why is the == True required?

Reply:
correct, it is redundant

Reply:
In the other places I have fixed it correctly, speed of light, you know ;)